c_emulator/riscv_sim.c

```diff
@@ -472,18 +494,16 @@ void tick_spike()
 void init_sail_reset_vector(uint64_t entry)
 {
 #define RST_VEC_SIZE 8
-  uint32_t reset_vec[RST_VEC_SIZE] = {
-    0x297,                                      // auipc  t0,0x0
-    0x28593 + (RST_VEC_SIZE * 4 << 20),         // addi   a1, t0, &dtb
-    0xf1402573,                                 // csrr   a0, mhartid
-    is_32bit_model() ?
-      0x0182a283u :                             // lw     t0,24(t0)
-      0x0182b283u,                              // ld     t0,24(t0)
-    0x28067,                                    // jr     t0
-    0,
-    (uint32_t) (entry & 0xffffffff),
-    (uint32_t) (entry >> 32)
-  };
+  uint32_t reset_vec[RST_VEC_SIZE]
```

Comment:
Also not great - I'll see if telling clang to align trailing comments makes this look better.

Reply:
If not, adding //clang-format off might be required here.